core/client-internal/src/main/java/alluxio/client/file/options/GetStatusOptions.java

```diff
@@ -21,36 +24,16 @@
 @PublicApi
 @NotThreadSafe
 public final class GetStatusOptions {
-  /** Whether or not to check the ufs if the path does not exist in Alluxio. */
-  private boolean mCheckUfs;
-
   /**
-   * @return a default {@link GetStatusOptions} based on the client's configuration
+   * @return the default {@link GetStatusOptions}
    */
+  @SuppressFBWarnings("ISC_INSTANTIATE_STATIC_CLASS")
```

Comment:
good point, it seems that they are not needed because the class has the toString() method